www/kubl05/SP1/css/styles.css

```diff
@@ -434,7 +502,39 @@ iframe {
         display: none;
     }
 
-    footer, footer p, footer img {
+    main {
+        background-color: white;
```

Comment:
chce to mensi pismo a mensi obrazky pro tisk

Reply:
ok